extensions/amp-fx-collection/0.1/test/integration/test-amp-fx-fly-in.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isExperimentOn, toggleExperiment} from '../../../../../src/experiments';
+
+const config = describe.configure().ifNewChrome();
+config.run('amp-fx-collection', function() {
+  this.timeout(100000);
```

Comment:
Double checking that this 100 second timeout is intentional?

Reply:
Yes, this one is intentional. It sets the global test timeout to be larger. Otherwise the default total timeout is 2000ms. One test alone exceeds that time.

Reply:
Looks like position observer integration test has the same timeout. It should be ok.

https://github.com/ampproject/amphtml/blob/63b4097da84f8e1563edc29c92b392f7f32d3153/test/integration/test-position-observer.js#L20-L23